src/cgame/rocket/rocket.cpp

```diff
@@ -486,30 +486,32 @@ void Rocket_Update()
 	Rml::Lua::Timer::Update(rocketInfo.realtime);
 }
 
-static std::string CG_EscapeHTMLChar( const char c, bool reuse = true, bool linebreak = false )
-{
-	std::unordered_map<char, std::string> htmlEscapeStrings;
-
-	htmlEscapeStrings['<'] = "&lt;";
-	htmlEscapeStrings['>'] = "&gt;";
-	htmlEscapeStrings['&'] = "&amp;";
-	htmlEscapeStrings['"'] = "&quot;";
-	htmlEscapeStrings['\''] = "&apos;";
-
-	if ( linebreak )
-	{
-		htmlEscapeStrings['\n'] = "<br />";
-	}
+static std::unordered_map<char, std::string> htmlEscapeStrings = {
+	{ '<', "&lt;" },
+	{ '>', "&gt;" },
+	{ '&', "&amp;" },
+	{ '"', "&quot;" },
+	{ '\'', "&apos;" },
+};
 
-	std::string htmlEscapedString = htmlEscapeStrings[ c ];
+static std::string CG_EscapeHTMLChar( const char c, bool reuse = true, bool endline = false )
```

Comment:
Constructing a std::string for every single character is going to make the code slower. There's nothing wrong with having a few lines of "duplicated" code in cases where the code is simple and rarely/never needs to be changed.